content/en/getting_started/tracing/_index.md

```diff
@@ -70,7 +70,7 @@ To create an application to observe in Datadog:
 
 To set up Datadog APM without needing to modify your application's code or the deployment process, use Single Step APM Instrumentation:
 
-<div class="alert alert-info"><strong>Note</strong>: <a href="https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/single-step-apm/">Single Step APM Instrumentation</a> is in beta. Alternatively, you can set up APM using <a href="https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/">Datadog tracing libraries</a>.</div>
+<div class="alert alert-info"><strong>Note</strong>: <a href="https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/single-step-apm/">Single Step APM Instrumentation</a> Alternatively, you can set up APM using <a href="https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/">Datadog tracing libraries</a>.</div>
```

Comment:
```suggestion
```
Actually since its no longer in beta we can remove this call-out and add the text above.